src/vendor/Template.js

```diff
@@ -1,11 +1,17 @@
+/* global require */
+
 import { array, coalesce, isArray, isString, missing } from './utils';
-import { toPairs } from './vectors';
-import { Log } from './logs';
 import { Data, isData } from './datas';
 import strings from './strings';
 
 let expandAny = null;
 
+class Template {
+  constructor(template) {
+    this.template = template;
+  }
+}
+
```

Comment:
I have been writing code that has import cycles: This mostly, works, but it will eventually prevent tests from passing as node gets too confused.  It also spews warnings in the browser console.  I had to fix them, which means moving definitions around.